jinja2/filters.py

```diff
@@ -685,9 +685,9 @@ def do_format(value, *args, **kwargs):
     return soft_unicode(value) % (kwargs or args)
 
 
-def do_trim(value):
-    """Strip leading and trailing whitespace."""
-    return soft_unicode(value).strip()
+def do_trim(value, chars):
```

Comment:
missing default value, this will break e.g. with StrictUndefined and even with the default Undefined it will most likely not strip anything when called without arguments.